src/PagesBody/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,43 @@
+import propTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+import MainButton from '../../Atoms/MainButton/MainButton';
+import ProfileCard from '../../Molecules/ProfileCard/ProfileCard';
+
+const ProfilePage = ({ userId }) => {
+  const navigate = useNavigate();
+
+  return (
+    <ColumnWrapper paddingLeftRight={1}>
+      <ProfileCard />
+      <MainButton
+        text="Twoje Psy"
+        onClick={() => navigate('/userDogs')}
+        ternary
+      />
+      <MainButton
+        text="Twoje Konkursy"
+        onClick={() => navigate(`./${userId}/contests`)}
+        // onClick={() => navigate('/contests')}
+        ternary
+      />
+      <MainButton
+        text="Twoje Dane"
+        onClick={() => navigate('/userData')}
+        ternary
+      />
+      <MainButton
+        text="Nadchodzące Konkursy"
+        onClick={() => navigate('/upcoming')}
+        secondary
+      />
```

Comment:
Links rather than buttons, something to consider

Reply:
Please, could You tell me why link ? We decided that if there a button it should behave like a button. Ofcourse its not a problem but i would love to know what would be a difference.

Reply:
That is a semantic issue really. Search on discussions about anchor tag vs. button. This one for example. https://dev.to/rahxuls/semantic-html-button-vs-anchor-5cnj Anything that takes you to another page - link. Triggers action? Button. You can style it the same, but keeping the semantics correct would be good.

Reply:
thanks